src/main/java/stringcalculator/StringCalculator.java

```diff
@@ -0,0 +1,58 @@
+package stringcalculator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class StringCalculator {
+    public static final String DELIMITER = " ";
+
+    public static int calculate(String equation) {
+        List<String> targets = splitEquation(equation);
+        validateTargetList(targets);
+
+        int result = stringToInt(targets.get(0));
+        for (int i = 1; i < targets.size(); i += 2) {
+            validateString(targets.get(i));
+            result = calculateOneOperation(result, targets.get(i), stringToInt(targets.get(i+1)));
+        }
+        return result;
+    }
+
+    private static void validateTargetList(List<String> targets) {
```

Comment:
```suggestion
    private static void validateTargets(List<String> targets) {
```
로 List를 명시할 필요없이 복수형으로 나타내는 것이 더 명시적이지 않을까요?